vent/types/sql_table.go

```diff
@@ -31,19 +31,37 @@ type UpsertColumn struct {
 	UpdPosition int
 }
 
-// defined fixed sql column names in log and event tables
+// SQL log & dictionary tables
+const SQLLogTableName = "_vent_log"
+const SQLDictionaryTableName = "_vent_dictionary"
+
+// defined fixed sql column names in log table
 const (
-	SQLColumnNameHeight      = "_height"
-	SQLColumnNameIndex       = "_index"
-	SQLColumnNameTxHash      = "_txhash"
-	SQLColumnNameEventName   = "_eventname"
-	SQLColumnNameEventType   = "_eventtype"
-	SQLColumnNameEventFilter = "_eventfilter"
-	SQLColumnNameTableName   = "_tablename"
-	SQLColumnNameTimeStamp   = "_timestamp"
+	//log
 	SQLColumnNameId          = "_id"
+	SQLColumnNameTimeStamp   = "_timestamp"
+	SQLColumnNameEventName   = "_eventname"
 	SQLColumnNameRowCount    = "_rowcount"
+	SQLColumnNameEventFilter = "_eventfilter"
+	SQLColumnNameHeight      = "_height"
+
+	//common
+	SQLColumnNameTableName = "_tablename"
+
+	//dictionary
+	SQLColumnNameColumnName   = "_columnname"
+	SQLColumnNameColumnType   = "_columntype"
+	SQLColumnNameColumnLength = "_columnlength"
+	SQLColumnNamePrimaryKey   = "_primarykey"
+	SQLColumnNameColumnOrder  = "_columnorder"
+
+	//auxiliary not in DB
+	SQLColumnNameIndex     = "_index"
+	SQLColumnNameTxHash    = "_txhash"
+	SQLColumnNameEventType = "_eventtype"
 )
 
-// SQL log table
-const SQLLogTableName = "_vent_log"
+// SQL dictionary table
+
+// defined fixed sql column names in dictionary
+const ()
```

Comment:
Is this something that is not complete yet?

Reply:
removed